api/docs/release.dox

```diff
@@ -65,6 +65,10 @@ The following are part of the DynamoRIO release distribution:
   PR 225255: list profile library too
 \endif
 
+ - An additional parameter in the accessors to the cache stats that passes
```

Comment:
This is not in the right place: the list here is a list of key files in the tarball.  It would go in the list on line 169 of "minor compatibility changes".  Also, generally we append to these lists, not prepend.

Reply:
Done.